index.d.ts

```diff
@@ -85,6 +91,7 @@ declare class Emittery {
 	@returns An unsubscribe method.
 	*/
 	on(eventName: EventName, listener: (eventData?: unknown) => void): Emittery.UnsubscribeFn;
+	on(eventName: typeof Emittery.listenerAdded | typeof Emittery.listenerRemoved, listener: (eventData: EventListenerData) => void): Emittery.UnsubscribeFn
```

Comment:
Can you add a type test to ensure this overload works correctly?

Reply:
Fixed!